demo/site/tsconfig.server.json

```diff
@@ -8,5 +8,5 @@
         "isolatedModules": false,
         "noEmit": false
     },
-    "include": ["server.ts", "tracing.ts"]
+    "include": ["server.ts", "tracing.ts", "cache-handler.ts"]
```

Comment:
The cache handler isn't really part of the server. But we can neglect this in favor of simplicity, I guess.